lumibot/data_sources/yahoo.py

```diff
@@ -96,3 +100,13 @@ def _pull_source_bars(self, symbols, length, timestep=MIN_TIMESTEP, timeshift=No
     def _parse_source_symbol_bars(self, response, symbol):
         bars = Bars(response, self.SOURCE, symbol, raw=response)
         return bars
+
+    @contextmanager
```

Comment:
I believe you are searching for the option `progress=False` to silence the printed message...